lib/report_portal/cucumber/parallel_report.rb

```diff
@@ -55,6 +64,8 @@ def initialize(desired_time)
             ReportPortal.launch_id = f.read
             f.flock(File::LOCK_UN)
           end
+          sleep_time = ENV['TEST_ENV_NUMBER'].to_i
+          sleep(ENV['TEST_ENV_NUMBER'].to_i)
```

Comment:
please add description why do we need the sleep?
and use sleep_time variable.

Reply:
Will do!